src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -87,6 +80,63 @@ private void VerifyTypeIL(CSharpCompilation compilation, string typeName, string
             CompileAndVerify(compilation).VerifyTypeIL(typeName, expected);
         }
 
+        [Fact]
+        public void TestVirtualPropertyOverride()
```

Comment:
>TestVirtualPropertyOverride [](http://example.com/codeflow?start=20&length=27)

Please add a test where SemanticModel is the first one to bind the accessor. Are we going to get any diagnostics after that?